packages/neovim-git/neovim-git.pacscript

```diff
@@ -1,21 +1,81 @@
 pkgname="neovim-git"
 gives="neovim"
+pkgver="0.10.1"
+repology=("project: ${gives}")
 source=("https://github.com/neovim/neovim.git")
-pkgver="0.9.1"
 url='https://neovim.io/'
 pkgdesc="Vim fork maintained by the community"
 maintainer=("Elsie19 <elsie19@pm.me>")
 makedepends=("ninja-build" "gettext" "libtool-bin" "autoconf" "automake" "cmake" "g++" "pkg-config" "unzip")
+replaces=("neovim" "nvim" "neovim-runtime" "neovim-git" "neovim-app" "neovim-deb" "neovim-bin")
+conflicts=("${replaces[@]}")
 external_connection=true
-breaks=("neovim-bin" "neovim" "neovim-app" "neovim-deb")
 
 build() {
-  cd "${_archive}"
+  cd "${gives}"
   make CMAKE_BUILD_TYPE=Release CMAKE_INSTALL_PREFIX=/usr -j"${NCPU}"
 }
 
 package() {
-  cd "${_archive}"
+  cd "${gives}"
   make install DESTDIR="${pkgdir}"
+  install -Dm644 "LICENSE.txt" -t "${pkgdir}/usr/share/licenses/${gives}"
+  install -Dm644 "README.md" -t "${pkgdir}/usr/share/doc/${gives}"
+  install -Dm755 /dev/stdin -t "${pkgdir}/usr/libexec/${gives}/ex" << 'EOF'
```

Comment:
`-t` is incorrect to use here, as it takes a directory as an argument. So this will create `/usr/libexec/${gives}/ex/stdin`

Reply:
D'oh! Good catch!